docs/development/pigs/pig-011.rst

```diff
@@ -0,0 +1,138 @@
+everywhere (instead of a mix of OrderedDict and dict)
+
+The first point is the most important one: the cost of supporting Python 3.5
+isn't very high: if you want or need it, let us know!
+
+Sunpy has already dropped Python 3.5 support in v1.0. Astropy still supports
+Python 3.5, they will move to 3.6+ with Astropy 4.0 in early 2010.
+
+tbd: decide on min version for other dependencies
+
+Distribution
+============
+
+Most Gammapy users use ``conda`` and binaries created via ``conda-forge``.
+Sufficient or do we want to support other distribution channels?
+
+Wheels
+------
+
+The wheel binary package format was introduced as a standard for Python packages
+in 2012 in `PEP 427`_. Most Python projects create and distribute wheels (see
+https://pythonwheels.com), including packages much more complex than Gammapy
+like Numpy, Scipy, Tensorflow.
+
+At this time, we don't plan to offer wheels for Gammapy, but to only distribute
+source distributions ``sdist`` at https://pypi.org/project/gammapy/ .
+
+The reason is that binary wheels need to be built for each supported platform,
+and they need to be distributed via PyPI, i.e. to create them a set of scripts
+or continuous delivery configuration has to be set up and maintained by the
+package maintainers. This is contrary and more difficult to achieve than binary
+conda packages, where conda-forge exists as a separate build infrastructure and
+tooling that starts by taking the ``sdist`` from PyPI and then uploading the
+conda packages to anaconda.org.
+
+At this point, nothing as simple as conda-forge has emerged for wheels, so
+different projects or small group of projects set up their own system, e.g
+https://github.com/MacPython/astropy-wheels for Astropy or
+https://iscinumpy.gitlab.io/post/azure-devops-python-wheels/ for iminuit and the
+scikit-hep packages.
+
+This hasn't happened for the Astropy affiliated packages (or all Python
+packages) and we in the Gammapy team don't have the expertise or time to
+contribute to that at this point. In the future, if a "wheel-forge" emerges, we
+can start supporting wheels for Gammapy.
+
+Actually, there is one platform where binary wheels for Gammapy and all
+dependencies exits already. If you ``pip install gammapy`` on Raspberry Pi,
+you'll get binary wheels thanks to this guy:
+https://www.raspberrypi.org/blog/piwheels/
+
+Macports
+--------
+
+We suggest to drop support for Macports, i.e. deleted the `Installation with
+Macports`_ page from the installation instructions.
+
+As can be seen in the `Gammapy in Macports`_ history, the last update was to
+Gammapy v0.7 in March 2018, and not a single user has complained that Gammapy
+wasn't updated since.
+
+It could be that there is still interest and Macports users, and they e.g. only
+use it to install Python and our dependencies, and then on top of that install
+Gammapy with pip. If you use Macports and want ``port install py37-gammapy``,
+let us know!
+
+Others
```

Comment:
Does Ole package gammapy for Debian? I believe most/all affiliated packages are included?

Reply:
I'll clarify Debian and other distros for Gammapy, See #324